test/draw/RasterRgbaTest.ooc

```diff
@@ -16,6 +16,15 @@ RasterRgbaTest: class extends Fixture {
 	sourceFlower := "test/draw/input/Flower.png"
 	init: func {
 		super("RasterRgbaTest")
+		this add("Test file not found", func {
+			try {
+				image := RasterRgba open("nonExistingFile")
+				expect(false)
+				expect(image, is equal to (null as RasterRgba))
+			} catch (e: Exception) {
+				expect(e message contains("Failed to load image: "))
```

Comment:
I agree that testing for a specific string isn't optimal, but to test that an exception is triggered will trigger on any type of exception and not necessary due to the failure we want to test. Ideally I guess you would want a separate exception type and trigger on that